engine/src/main/java/org/pentaho/di/job/entries/ping/JobEntryPing.java

```diff
@@ -245,6 +245,11 @@ public String getRealHostname() {
   }
 
   public String getTimeOut() {
+    if ( !isNumeric( timeout ) ) {
+      logMinimal( BaseMessages.getString( PKG, "JobPing.WarningOnlyNumericValue",
```

Comment:
I don't understand what this logMinimal() statement is trying to print out. Can you give an example of what is will produce?